contents/docs/features/actions.md

```diff
@@ -16,101 +16,99 @@ For example, a typical action might be one of the following:
+This will open the following page:
 
-* Frontend Element
-* Custom Event
-* Page view
+![](../../images/create-action.png)
+<br>
 
-![](../../images/03/Posthog-19.png)
+Here, you have a choice of two options:
 
-Page views can match urls containing a string or match exactly.
+#### Inspect element on your site
 
-![](../../images/03/Posthog-20.png)
+This will open the domain you provided to PostHog so that you can select an element with our Inspect tool and create an action from this.
 
-Custom Events can be sent to your PostHog instance by API we have several libraries to allow you to do [this](/Integrations).
+> **Note:** Our [Toolbar](/docs/features/toolbar) is making this significantly easier.
 
-Frontend Elements can be selected directly on your site, click 'Select element on site'.
+#### From event or pageview
 
-![toolbar pop up](../../images/03/Posthog-21.png)
+Alternatively you can create an event based on `pageview` or an existing [event](/docs/features/events) on your website.
 
-This will pop open the domain you provided in Setup.
+Choosing this option will open the following page where you can create your desired Action:
 
-You will see a PostHog toolbar pop up on the right hand side:
+![](../../images/action-from-event.png)
+<br>
 
-![PostHog Toolbar](../../images/02/Screenshot-2020-02-09-at-14.48.17.png)
+ 
+#### Note: Actions Containing Multiple Events 
 
+It is possible for an action to match multiple events. To do this, click ‘Add another match group’ when you are creating your action (see picture above).
 
-## Labelling events with the toolbar
 
-In this example, we may want to track users clicking ‘Sign up!’ to our example movie streaming website, HogFlix.
+## Action Types
 
-Use the toolbar to create a name for the Action. Pro-tip: the best convention for this is ‘Object – Verb’, but it’s up to you.
+As shown above, there 3 types of Actions you can record:
 
-Next, click ‘Inspect element’ to tag the element on the page that you want to grab events relating to. Your application screen will be highlighted in blue.
+#### Page View
 
-Now, move your mouse over the element in question. You’ll see PostHog highlight the element you are hovering over, and the toolbar will populate with the relevant fields as you move the mouse around.
+Page views can match urls containing a string or match exactly.
+
+![](../../images/03/Posthog-20.png)
+<br>
 
-![PostHog toolbar in action](../../images/02/Screenshot-2020-02-09-at-14.51.31.png)
 
-## Example - labelling a sign up Action
+#### Frontend Element
 
-In this example, we may want to track users clicking ‘Sign up!’ to our example movie streaming website, HogFlix.
+Frontend Elements are actions based on some element on your website, such as a button or an input. The easiest way to select them is with the 'Inspect element on your site' functionality described above. However, you can also set them manually if you wish by providing something to identify the element, like a selector.
```

Comment:
We don't grab from every element type automatically. For example, if ```div``` is used as a psuedo button, then this will fail. It would be worth just listing the element types we auto track here (you may need to look in code or ask)